### ISSUE TITLE
Introduce Returnable Transport Items

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#1543]
When we introduced the deposit-refund system, we added the concept into the e-commerce layer.
For example, the `Order` class, which represents an e-commerce order, has properties like `reusablePackagingEnabled`.
The mistake is that the deposit-refund system, while linked with food ordering, actually belongs to logistics, not e-commerce. 

In logistics, it seems to be named [Returnable Transport Items](https://www.barcoding.com/blog/the-value-of-returnable-transport-items/) or [Returnable Transport Packaging](https://help.sap.com/saphelp_me60/helpdata/EN/40/62bd534f22b44ce10000000a174cb4/content.htm?no_cache=true).

The main idea between this, is that messengers, when completing a task, should also be able to grab RTIs, and that those RTIs should be traceable. It basically means that RTIs should be moved/attached to the `Task` class. A `Task` can have 0-N RTIs. 

---

It's still not 100% clear, but the goal is to cover at least those 2 use-cases:

#### 1/ Double-checking "expected" returnable packaging

This is what is needed for deposit-refund system for restaurants. A customer has some packages (from a previous order) at home, and wants to give them to the messenger for the next order. In this case, the messenger should be presented with a sort of "confirmation" screen when completing the dropoff task. 
Basically, it is like saying: "the customer said he/she would give back 2 packagings. Do you confirm he/she gave you 2 packagings?"

#### 2/ Being able to add manually "unexpected" returnable packaging

Let's say that a company is delivering supermarket groceries. 
The supermarket, because it is organic & zero-waste, only uses glass bottles for milk. 
Customers who have previously bought glass bottles of milk may give to the messenger those glass bottles, when they received their next order. 
In this case, the messenger should be able, when completing the task, to manually add X bottles of glass.

**Comment by ERAMANrepartos** [comment]
@alexsegura can this be closed?
